### ISSUE TITLE
Master Branch - Nokogiri::VERSION = 1.7.0.1

### ISSUE THREAD

**Issue by MSP-Greg** [sparklemotion/nokogiri#1642]
[Nokogiri::VERSION = 1.7.0.1](https://github.com/sparklemotion/nokogiri/blob/master/lib/nokogiri/version.rb#L3)

Might this be changed in the master branch to either reflect the current release version or the next?  I'd submit a PR, but I don't know which is preferred...

**Comment by flavorjones** [comment]
Thanks for asking this question. The short answer is: no.

The long answer is: `v1.7.1` and `v1.7.2` are on a separate branch off `v1.7.0.1` (commit e1f2c2d). If you follow the history of `master`, you'll see it does not contain `v1.7.1` or `v1.7.2`. Therefore its most recent parent is `v1.7.0.1`, and the version info in that file is accurate (or at least not misleading or confusing, which setting it to `v1.7.1` would be).

And honestly, between releases, I'm not overly concerned with the value of this constant.

I'm curious why you care what this value is? If the answer is "I'm shipping my code based on Nokogiri master", then I'd ask you to read https://github.com/sparklemotion/nokogiri/blob/master/Y_U_NO_GEMSPEC.md for context. We can easily cut an RC if that's what you need.

**Comment by MSP-Greg** [comment]
@flavorjones 

Long answer: Been using Nokogiri for quite a while, and had it doc'd locally, including the c code.  I've got ruby, rails, and AWS on a site, and decided to add Nokogiri master, although I haven't added it to main code that auto-updates the site.

Regardless, it seemed a little odd to view docs for master and see a [version number](https://msp-greg.github.io/nokogiri/Nokogiri.html#VERSION-constant) that is two behind current (1.7.2).

But, I understand your answer, and thanks.

Misc - I added the tutorials to the 'docs' list, and although my code is based on YARD, it picks up many of the RDoc declarations in c code, it shows 'nodoc' items as 'don't use', etc...
